sys/arduino/base.cpp

```diff
@@ -72,6 +72,11 @@ unsigned long micros()
     return xtimer_now_usec();
 }
 
+unsigned long millis()
+{
+    return xtimer_now_usec() / 1000;
```

Comment:
You could use `US_PER_MS` instead of hardcoded value 1000. I think @smlng will agree :)

Reply:
With the PR being small, I decided to squash right away. And I added a second commit to also update delay to use `US_PER_MS` as well.